ui/site/src/component/assets.ts

```diff
@@ -39,7 +39,7 @@ export const loadScript = (url: string, opts: AssetUrlOpts = {}): Promise<void>
 
 export const loadModule = (name: string): Promise<void> => loadScript(jsModule(name));
 export const loadIife = async (name: string, iife: keyof Window) => {
-  await loadModule(name);
+  loadModule(name);
```

Comment:
if we don't await this promise, is `window[iife]` already available?

Reply:
This looks like a vestige of an alternative solution to `window` globals not being set where I inserted a function in the `loadModule` callback chain to consolidate the `(window as any).blah = blah changes` in one place.  But I decided to go with a simpler but a bit messier approach and just set them directly in the module code.  Similar to the arrow function thing, looks like it was not fully backed out.  Good catch, `keyboardMove` dependents thank you.